api/src/utils/redact.ts

```diff
@@ -88,17 +94,33 @@ export function redact(input: UnknownObject, paths: Paths, replacement: string):
 }
 
 /**
- * Extract values from Error objects for use with JSON.stringify()
+ * Replace values and extract Error objects for use with JSON.stringify()
  */
-export function errorReplacer(_key: string, value: unknown) {
-	if (value instanceof Error) {
-		return {
-			name: value.name,
-			message: value.message,
-			stack: value.stack,
-			cause: value.cause,
-		};
-	}
+export function getReplacerFn(replacement: string, valuesToRedact?: string[]) {
+	const lowercasedValuesToRedact = valuesToRedact?.filter(v => typeof v === "string").map((v) => v.toLowerCase());
```

Comment:
```suggestion
	const lowercasedValuesToRedact = valuesToRedact?.filter(v => typeof v === "string" && v.length > 0).map((v) => v.toLowerCase());
```
we'll want to ignore 0-length strings too.
![image](https://github.com/directus/directus/assets/9389634/6bb1b3da-70f4-4a4f-94aa-70810129a07e)